app/ImageAnnotation.php

```diff
@@ -19,6 +19,13 @@ class ImageAnnotation extends Annotation
         'points' => 'array',
     ];
 
+    /**
+     * The attributes that should be included in the JSON response.
+     *
+     * @var array<int, string>
+     */
+    protected $appends = ['labelBOTLabels'];
+
```

Comment:
Maybe this should only be appended in the single response from the `store()` controller method. Otherwise it will be included anywhere annotations are used even if it has nothing to do with LabelBOT. You can use `$annotation->append('labelBOTLabels')` at runtime to do this.